### DISCUSSION TITLE
How do i remove the titles from the programs in taskbar? I want to show only icons

### DISCUSSION THREAD

**Discussion by Kauan9196** [valinet/ExplorerPatcher#721] [General]
How do i remove the titles from the programs in taskbar? I want to show only icons
![icon](https://user-images.githubusercontent.com/98352798/150853259-1933d7e5-b3ca-4b16-9fad-c0fec3bf5033.jpeg)

**Comment by chip33** [comment]
Hi,
it is done by setting "Combine taskbar icons on taskbar" to "Always combine"
![Screenshot 2022-01-24 213305](https://user-images.githubusercontent.com/48499740/150868951-9c47b7bf-4698-4f09-ace1-ccf1588ce0e7.png)

**Reply by Kauan9196** [reply]
it works, thank you

**Reply by chip33** [reply]
Great! you are welcome
